security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,51 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1-15-0"]
+== {cert-manager-operator} 1.15.0
+
+Issued: 2025-01-22
+
```

Comment:
@bharath-b-rh will share us a few advisory links, which we can add tomorrow.

Reply:
@subhtk , @swghosh Below are the advisory links created for the release.
- https://access.redhat.com/errata/RHEA-2025:0487 (cert-manager-operator bundle)
- https://access.redhat.com/errata/RHSA-2025:0535 (cert-manager-operator)
- https://access.redhat.com/errata/RHSA-2025:0536 (cert-manager)